tests/unit/delete-column-step-form.spec.ts

```diff
@@ -122,8 +109,8 @@ describe('Delete Column Step Form', () => {
       store,
       localVue,
     });
-    wrapper.setData({ column: 'columnB' });
-    await wrapper.find(WidgetAutocomplete).trigger('input');
+    wrapper.find(WidgetAutocomplete).vm.$emit('input', 'columnB');
```

Comment:
I think that this piece of code is inn a a test that checks something already tested in column-picker.spec.ts